### PR TITLE
worked on the media queries

### PR DESCRIPTION
**Pull request by Mwanasha-Omari**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by Cynthia-Matsitsa — APPROVED** [review]

**Review by Shalom-uwamahoro — APPROVED** [review]

**Review by Aber-Racheal — APPROVED** [review]

File: informational/src/Landing-page/index.css [hunk 1]
```diff
@@ -38,7 +35,7 @@ a{
 }
 .landing{
     position: relative;
-    background-image:  linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Documents/Informational-dev/Icons-Informational/informational/public/Images/man.jpeg");
+    background-image:  linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Icons-Informational/informational/public/Images/man.jpeg");
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:38]: This image path will create bugs when hosted

File: informational/src/Landing-page/index.css [hunk 2]
```diff
@@ -138,7 +140,7 @@ a{
         font-size: 10px;
     }
     .landing{
-        background-image:linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("/home/studen/Documents/Informational-dev/Icons-Informational/informational/public/Images/man.jpeg");
+        background-image:linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("/home/studen/Icons-Informational/informational/public/Images/man.jpeg");
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:143]: Same as first comment

File: informational/src/Landing-page/index.css [hunk 3]
```diff
@@ -239,7 +241,7 @@ a{
         font-size: 10px;
     }
     .landing{
-        background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Documents/Informational-dev/Icons-Informational/informational/public/Images/man.jpeg");
+        background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Icons-Informational/informational/public/Images/man.jpeg");
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:244]: Same

File: informational/src/Landing-page/index.css [hunk 4]
```diff
@@ -340,7 +342,7 @@ a{
...
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:345]: Same

File: informational/src/Landing-page/index.css [hunk 5]
```diff
@@ -442,7 +444,7 @@ a{
         font-size: 15px;
     }
     .landing{
-        background-image:linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("/home/studen/Documents/Informational-dev/Icons-Informational/informational/public/Images/man.jpeg");
+        background-image:linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("/home/studen/Icons-Informational/informational/public/Images/man.jpeg");
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:447]: Same

File: informational/src/Landing-page/index.css [hunk 6]
```diff
@@ -546,7 +548,7 @@ a{
         font-size: 20px;
     }
     .landing{
-        background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Documents/Informational-dev/Icons-Informational/informational/public/Images/man.jpeg");
+        background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url("/home/studen/Icons-Informational/informational/public/Images/man.jpeg");
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:551]: same

File: informational/src/Landing-page/index.css [hunk 7]
```diff
@@ -652,7 +654,7 @@ a{
...
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:657]: Same

File: informational/src/Landing-page/index.css [hunk 8]
```diff
@@ -760,7 +762,7 @@ a{
...
```
Comment (hunterachieng) [inline informational/src/Landing-page/index.css:765]: Same

**Pull request closed without merging**